web/src/settings_panel_menu.js

```diff
@@ -53,6 +53,14 @@ export class SettingsPanelMenu {
         this.$main_elem.on("click", "li[data-section]", (e) => {
             const section = $(e.currentTarget).attr("data-section");
 
+            /* We use an additional parameter in the hash of Users section
+               to display different sections.
+            */
+            if (section === "user-list-admin") {
+                browser_history.update_hash_internally_if_required(
+                    `#${this.hash_prefix}${section}/active-users-admin`,
```

Comment:
Can this be removed by just passing `right_section = "active-users-admin"` to `activate_section_or_default` and then just update the hash in `activate_section_or_default` for all cases including the case when `section === "user-list-admin"`.

Reply:
Sure , updated. 
Though I am updating the hash in `get_valid_users_section_panel` rather than `activate_section_or_default` because it is possible to get the `right_section` as undefined in `activate_section_or_default` but it will never be the case for `get_valid_users_section_panel`.